TusStorageHandler/minio_client.py

```diff
@@ -0,0 +1,54 @@
+"""Minio Client Configuration Module.
+
+It initializes a Minio client using environment variables
+loaded via dotenv. It also provides a function to create a new Minio
+bucket if it does not already exist.
+
+Usage:
+    - Ensure environment variables are set in a .env file:
+      - MINIO_HOSTNAME
+      - MINIO_PORT
+      - MINIO_ACCESS_KEY
+      - MINIO_SECRET_KEY
+      - MINIO_IS_SECURE (optional, defaults to 'true')
+      - MINIO_BUCKET_NAME
+
+    - Import `minio_client` to access:
+      - `minio_client`: Initialized Minio client instance
+      - `create_bucket()`: Function to create a new bucket if not exists.
+"""
+
+import os
+
+from dotenv import load_dotenv
+from minio import Minio
+
+load_dotenv()
+
+MINIO_HOSTNAME = os.getenv("MINIO_HOSTNAME")
```

Comment:
I would get these in a seperate file like consts..py or env.py just so they are easier to document and then import it whereever needed.